Makefile

```diff
@@ -44,17 +60,13 @@ docker-shell:
 # Targets for orchestrating project build
 #
 
-# Build Drupal and front-end using Docker containers
-build-docker:
-	./scripts/make/build-docker.sh
-
 # Initialise Apache Solr core in service container for Apache Solr (without Search API).
-build-docker-solr:
-	make docker-shell HOST=$(SOLR_HOST) COMMAND='make core=core1 config_set=apachesolr -f /usr/local/bin/actions.mk'
+init-solr:
+	make shell HOST=$(SOLR_HOST) COMMAND='make core=core1 config_set=apachesolr -f /usr/local/bin/actions.mk'
 
 # Initialise Apache Solr core in service container for Apache Solr (via Search API).
-build-docker-search:
-	make docker-shell HOST=$(SOLR_HOST) COMMAND='make core=core1 -f /usr/local/bin/actions.mk'
+init-searchapi-solr:
```

Comment:
I think we need to keep that language of "install, build, start" consistently across sub tasks, and here  this should probably be `build-solr`, because it's a necessary step to run before the application can be `start`ed

Reply:
agreed

Reply:
sage words